src/chat.rs

```diff
@@ -4166,4 +4169,65 @@ mod tests {
         )
         .await
     }
+
+    #[async_std::test]
+    async fn test_sticker_forward(filename: &str, bytes: &[u8], w: i32, h: i32) -> Result<()> {
```

Comment:
What these arguments are for? Normally tests don't have arguments, looks like `async_std::test` macro simply ignores them.